src/MSBuild/app.amd64.config

```diff
@@ -184,6 +184,17 @@
           <assemblyIdentity name="Microsoft.VisualStudio.CodeAnalysis.Sdk" culture="neutral" publicKeyToken="b03f5f7f11d50a3a" />
           <codeBase version="17.0.0.0" href="..\..\..\Microsoft\VisualStudio\v17.0\CodeAnalysis\Microsoft.VisualStudio.CodeAnalysis.Sdk.dll" />
         </dependentAssembly>
+
+        <!-- Redirects for SDK resolver components -->
```

Comment:
Can the https://github.com/dotnet/msbuild/blob/main/documentation/NETFramework-NGEN.md#microsoftdotnetmsbuildsdkresolver be linked here?
I was confused why the version is fixed here - so the doc can help clarify it.

Reply:
I was a bit hesitant about including a GitHub link in a file that gets placed on end user machines unscrubbed. Especially since this is not a perma-link. But it's potentially useful for posterity. Done, thank you!